src/omv/imlib/draw.c

```diff
@@ -3144,8 +3144,14 @@ void imlib_draw_image(image_t *dst_img,
             t_src_img.data = src_img->data;
         }
 
+        // Query available on-chip RAM.
         uint32_t size;
         void *data = fb_alloc_all(&size, FB_ALLOC_PREFER_SPEED | FB_ALLOC_CACHE_ALIGN);
```

Comment:
Perhaps we should add an fb_alloc_available() function.

Reply:
https://github.com/openmv/openmv/issues/2522

Reply:
How does the above query on chip RAM ? If there isn't enough it will just allocate from DRAM and return the size.

Reply:
FB_ALLOC_PREFER_SPEED it will allocate to overlay stack.